src/app/providers/authentication/authentication.ts

```diff
@@ -12,10 +12,12 @@ export enum Token {
 
 @Injectable()
 export class AuthenticationProvider {
-
   public authenticationSettings: any;
+
```

Comment:
Has this been introduced by the linter?

Reply:
Yes by the looks of things, I tried out the --fix when running eslint command